root/types/edit_types.js

```diff
@@ -0,0 +1,1095 @@
+    +artist_credit: ArtistCreditT,
+    +barcode: string | null,
+    +comment: string,
+    +events?: $ReadOnlyArray<ReleaseEventT>,
+    +language: LanguageT | null,
+    +name: string,
+    +packaging: ReleasePackagingT | null,
+    +release: ReleaseT,
+    +release_group: ReleaseGroupT,
+    +script: ScriptT | null,
+    +status: ReleaseStatusT | null,
+  },
+  +edit_type: EDIT_RELEASE_CREATE_T,
+}>;
+
+declare type AddReleaseGroupEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +comment: string,
+    +name: string,
+    +release_group: ReleaseGroupT,
+    +secondary_types: string,
+    +type: ReleaseGroupTypeT | ReleaseGroupHistoricTypeT | null,
+  },
+  +edit_type: EDIT_RELEASEGROUP_CREATE_T,
+}>;
+
+declare type AddReleaseLabelEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +catalog_number: string,
+    +label?: LabelT,
+    +release?: ReleaseT,
+  },
+  +edit_type: EDIT_RELEASE_ADDRELEASELABEL_T,
+}>;
+
+declare type AddRemoveAliasEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +[coreEntityType: CoreEntityTypeT]: CoreEntityT,
+    +alias: string,
+    +begin_date: PartialDateT,
+    +end_date: PartialDateT,
+    +ended?: boolean,
+    +entity_type: CoreEntityTypeT,
+    +locale: string | null,
+    +primary_for_locale: boolean,
+    +sort_name: string | null,
+    +type: AliasTypeT | null,
+  },
+  +edit_type: EDIT_ALIAS_ADD_T | EDIT_ALIAS_DELETE_T,
+}>;
+
+declare type AddSeriesEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +comment: string,
+    +name: string,
+    +ordering_type: SeriesOrderingTypeT | null,
+    +series: SeriesT,
+    +type: SeriesTypeT | null,
+  },
+  +edit_type: EDIT_SERIES_CREATE_T,
+}>;
+
+declare type AddStandaloneRecordingEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +comment: string | null,
+    +length: number | null,
+    +name: string,
+    +recording: RecordingT,
+    +video: boolean,
+  },
+  +edit_type: EDIT_RECORDING_CREATE_T,
+}>;
+
+declare type AddWorkEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +attributes?: {
+      [attributeName: string]: $ReadOnlyArray<WorkAttributeT>,
+    },
+    +comment: string,
+    +iswc: string,
+    +language?: LanguageT,
+    +languages?: $ReadOnlyArray<LanguageT>,
+    +name: string,
+    +type: WorkTypeT | null,
+    +work: WorkT,
+  },
+  edit_type: EDIT_WORK_CREATE_T,
+}>;
+
+declare type ChangeReleaseQualityEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +quality: CompT<QualityT>,
+    +release: ReleaseT,
+  },
+  edit_type: EDIT_RELEASE_CHANGE_QUALITY_T,
+}>;
+
+declare type ChangeWikiDocEditT = $ReadOnly<{
+  ...GenericEditT,
+  +display_data: {
+    +new_version: number | null,
+    +old_version: number | null,
+    +page: string,
+  },
+  +edit_type: EDIT_WIKIDOC_CHANGE_T,
+}>;
+
+declare type EditAliasEditT = $ReadOnly<{
+  ...GenericEditT,
+  +alias: AliasT | null,
```

Comment:
This being here (but not elsewhere) causes a flow error:

`Cannot create `EditNote` element because property `alias` is missing in  object type [1] but exists in  object type [2] in property `edit`.`